backend/src/common/constants/instance.constants.ts

```diff
@@ -0,0 +1,5 @@
+const INSTANCE_TYPE = 't2.micro';
+const MAX_WAITING_TIME = 90;
+const USERNAME = 'ec2-user';
```

Comment:
```suggestion
const INSTANCE_TYPE = 't2.micro';
const MAX_WAITING_TIME = 90;
const USERNAME = 'ec2-user';
```
collect all constants into enum `InstanceDefaultParam`